notebooks/data-sources/TestGrid/metrics/correlated_failures.ipynb

```diff
@@ -4,40 +4,53 @@
    "cell_type": "markdown",
```

Comment:
<p>If we add the "if automation then Ceph else disk" logic here, then this notebook would be directly ready for automation. We could do it in a later PR as well...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>